examples/gno.land/p/demo/auction/auction.gno

```diff
@@ -0,0 +1,94 @@
+package auction
+
+import (
+	"std"
+	"strconv"
+	"time"
+
+	"gno.land/p/demo/ownable"
+	"gno.land/p/demo/ufmt"
+)
+
+// Auction struct
```

Comment:
"Auction struct" is not particularly useful. Some general suggestions: https://go.dev/doc/comment#type

Reply:
[261c6992](https://github.com/gnolang/gno/pull/2265/commits/261c699234c04d239a8ab390c4d9b9683e51beeb)